packages/mongodb-chatbot-server/README.md

```diff
@@ -55,7 +55,6 @@ export const {
   OPENAI_ENDPOINT,
   OPENAI_API_KEY,
   OPENAI_EMBEDDING_DEPLOYMENT,
-  OPENAI_EMBEDDING_MODEL_VERSION,
   OPENAI_CHAT_COMPLETION_MODEL_VERSION,
```

Comment:
can delete this too?

Reply:
Not yet, we still reference it in our analyzer code. Ticket to update that https://jira.mongodb.org/browse/EAI-502